src/circular-progress.jsx

```diff
@@ -72,15 +72,17 @@ const CircularProgress = React.createClass({
     this._rotateWrapper(wrapper);
   },
 
+  componentWillUnmount() {
+    clearTimeout(this._scalePathTimer);
```

Comment:
I'm not sure about the `_`. I would say we can remove it. It's not shared to the public API, I don't see the need.